packages/cardano-services/src/Handle/TypeOrmHandleProvider.ts

```diff
@@ -55,4 +61,33 @@ export class TypeOrmHandleProvider extends TypeormProvider implements HandleProv
       return handles.map((handle) => mapEntity(entities.find((entity) => entity.handle === handle)));
     });
   }
+
+  async getPolicyIds(): Promise<Cardano.PolicyId[]> {
+    return this.withDataSource(async (dataSource) => {
+      const columnName = 'policy_id';
+      const fetchPolicyIdsFromDB = async () => {
+        const distinctPolicyIds = await dataSource
+          .getRepository(HandleEntity)
+          .createQueryBuilder('handle')
+          .select(`DISTINCT ${columnName}`, columnName)
+          .getRawMany();
+        const policyIds = distinctPolicyIds.map((row) => row[columnName]);
+
+        if (policyIds.length === 0) {
+          throw new Error('Value from the database is 0');
+        }
+
+        return policyIds;
+      };
+
+      try {
+        return this.inMemoryCache.get('policyIds', fetchPolicyIdsFromDB);
+      } catch (error: any) {
+        if (error.message === 'Value from the database is 0') {
```

Comment:
```suggestion
        if (error instanceof Error && error.message === 'Value from the database is 0') {
```

This should allow to not disable `@typescript-eslint/no-explicit-any` eslint rule